spirent/service/service.go

```diff
@@ -0,0 +1,64 @@
+package service
+
+import (
+	"github.com/SpirentOrion/luddite"
+	"os"
+)
+
+type Config struct {
+	Service luddite.ServiceConfig
+}
+
+type SpirentService struct {
+	Service luddite.Service
+	Started bool
+}
+
+var (
+	service *SpirentService
+)
+
+func Service() *SpirentService {
```

Comment:
what's the purpose for this service?

Reply:
Creates a REST endpoint that output (magellan) and also input (host_agent, etc) can register endpoints into.  This will allow telegraf to control without restarts.